tests/e2e/helpers/database.ts

```diff
@@ -116,6 +117,20 @@ export async function acceptLicenseTermsAndAddDatabase(databaseParameters: AddNe
     await myRedisDatabasePage.clickOnDBByName(databaseName);
 }
 
+/**
+ * Accept License terms and add database using api
+ * @param databaseParameters The database parameters
+ * @param databaseName The database name
+*/
+export async function acceptLicenseTermsAndAddDatabaseApi(databaseParameters: AddNewDatabaseParameters, databaseName: string): Promise<void> {
+    await acceptLicenseTerms();
+    await addNewStandaloneDatabaseApi(databaseParameters);
+    // Reload Page
+    await t.eval(() => location.reload());
```

Comment:
Reason?

Reply:
this is reload of page after adding database through the api - we need to reload a page to see the added through api item

Reply:
I think we discussed it in the past. Couldn't you make a while loop waiting until the DB is added?

Reply:
no, this is not the same situation
when DB added through the api request, it can't be displayed on UI without reloading the page